src/App.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Calculator from './components/Calculator';
+
+class App extends React.PureComponent {
```

Comment:
- [ ] I kindly suggest that you use make your `App` component a functional component instead of making use of class.